packages/devtools/src/lib/store.ts

```diff
@@ -0,0 +1,89 @@
+import { set as lodashSet, merge } from "lodash-es";
+import type {
+  CallToolResponse,
+  OpenAiProperties,
+  UnknownObject,
+} from "skybridge/web";
+import { create } from "zustand";
+
+export type OpenAiLog = {
+  id: string;
+  timestamp: number;
+  command: string;
+  args: UnknownObject;
+};
+
+type ToolData = {
+  input: Record<string, unknown>;
+  response: CallToolResponse;
+  openaiRef: React.RefObject<HTMLIFrameElement> | null;
+  openaiLogs: OpenAiLog[];
+  openaiObject: OpenAiProperties | null;
+};
+
+export type Store = {
+  selectedTool: string | null;
+  setSelectedTool: (tool: string | null) => void;
+  tools: {
+    [name: string]: ToolData;
+  };
+  setToolData: (tool: string, data: Partial<ToolData>) => void;
+  pushOpenAiLog: (tool: string, log: Omit<OpenAiLog, "id">) => void;
+  updateOpenaiObject: (
+    tool: string,
+    key: keyof OpenAiProperties,
+    value: unknown,
+  ) => void;
+};
+
+export const useStore = create<Store>()((set) => ({
```

Comment:
There is a typo here no? only createStore does not require any args
```suggestion
export const useStore = create<Store>((set) => ({
```